app/setup_script.sql

```diff
@@ -13,36 +13,36 @@ grant application role app_viewer to application role app_admin;
 -- so that we do not clobber state whenever we upgrade the app
 create schema if not exists config_data;
     grant usage on schema config_data to application role app_admin;
-    execute immediate from './config_data/configuration.sql';
+    execute immediate from './sql_lib/config_data-configuration.sql';
```

Comment:
FWIW I did prefer the previous organization in subdirectories. Why is this change needed?

Reply:
The manual upload (used in the quickstart) does not support recursive uploads, so it's just a torture to do that manually. That was a compromise until we change the tutorial to use snow cli (instead of just support it)